qiskit/test/utils.py

```diff
@@ -1,87 +0,0 @@
-# This code is part of Qiskit.
-#
-# (C) Copyright IBM 2017, 2018.
-#
-# This code is licensed under the Apache License, Version 2.0. You may
-# obtain a copy of this license in the LICENSE.txt file in the root directory
-# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
-#
-# Any modifications or derivative works of this code must retain this
-# copyright notice, and modified files need to carry a notice indicating
-# that they have been altered from the originals.
-
-"""Utils for using with Qiskit unit tests."""
-
-import logging
-import os
-from enum import Enum
-from itertools import product
-
-from qiskit import __path__ as qiskit_path
-
-
-class Path(Enum):
-    """Helper with paths commonly used during the tests."""
-
-    # Main SDK path:    qiskit/
-    SDK = qiskit_path[0]
-    # test.python path: qiskit/test/python/
-    TEST = os.path.normpath(os.path.join(SDK, "..", "test", "python"))
-    # Examples path:    examples/
-    EXAMPLES = os.path.normpath(os.path.join(SDK, "..", "examples"))
-    # Schemas path:     qiskit/schemas
-    SCHEMAS = os.path.normpath(os.path.join(SDK, "schemas"))
-    # Sample QASMs path: qiskit/test/python/qasm
-    QASMS = os.path.normpath(os.path.join(TEST, "qasm"))
```

Comment:
Only `SDK` was actually used (`TEST` was transitively dead - only used by a dead function), and even then only once.  Inlined, since it's trivial.